src/app/Main/Body/QueryPage/QueryPage-Layout.component.tsx

```diff
@@ -42,6 +42,7 @@ export function QueryPageLayoutComponent(props: {
 }): JSX.Element {
     const [drawerWidth, setDrawerWidth] = useState<number>(433);
     const [open, setOpen] = useState<boolean>(true);
+    const [sumRowCol, setSumRowCol] = useState<boolean>(true);
```

Comment:
Reject: Why is the state defined here? What has the QueryPageLayout to do with the show sums toggle mechanism? So because this feature is specific to the table it should be defined there.

Reply:
Done. Commit "Some more changes "

Reply:
Done. Commit "Some more changes"